Week2/homework/index.js

```diff
@@ -0,0 +1,114 @@
+console.log(1);
+let HiIndo='Hallo, dunia!'
+let HiIta='Ciao, mondo!' 
+let HiSpa='Hola, mundo!'
+let HiRus='Priviet, mir!'
+let HiKaz='Salamtsizba, duniye!'
+let HiDutch='Hallo, wereld!'
+let HiFr='Bonjour, le monde!'
+let HiTr='Merhaba, dunya!'
+let HiGr='Hallo, welt!'
+let HiRomanian='Salut, lume!'
+console.log(HiIndo);
+console.log(HiIta);
+console.log(HiSpa);
+console.log(HiRus);
+console.log(HiKaz);
+console.log(HiDutch);
+console.log(HiFr);
+console.log(HiTr);
+console.log(HiGr);
+console.log(HiRomanian);
+
+console.log(2);
+console.log('I'm awesome');
+// Uncaught SyntaxError: missing ) after argument list
+//The symbol used for the string is the same with the symbol used in the syntax of the string.
+// When I changed it into sth else, i gives what i wanted to write.
+console.log(" I'm awesome!");
+
+
+console.log(3)
+let x;
+x=3;
+
+console.log("the value of my variable x will be: 3");
```

Comment:
Ah, you also have to `console.log` the value of `x` before you initialize it. Can you also do that?